src/librustc_codegen_ssa/lib.rs

```diff
@@ -66,6 +66,7 @@ pub struct ModuleCodegen<M> {
     pub kind: ModuleKind,
 }
 
+pub const METADATA_FILENAME: &str = "rust.metadata.bin";
 pub const RLIB_BYTECODE_EXTENSION: &str = "bc.z";
```

Comment:
This is a problem, because `bc` is LLVM-specific.

Reply:
But feel free to file an issue and leave it for later.

Reply:
What is the convention on those kind of issues? File it right now and add a comment here referencing it?